### PR TITLE
Update feature templates

### PR DESCRIPTION
**Pull request by denis-fokin**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Substitute this text with a concise description of the problem. Please be clear what parts of the problem are considered to be in-scope and out-of-scope.
+
+_Example:_ Sometimes engine does not emit any tests. Seems that in such situations we can use a fuzzer.
+
+**Solution you'd like**
+
+Substitute the text with a clear and concise description of what you want to happen.
+
+_Example:_ We can significantly improve coverage with a smart fuzzer. We can not only feed the API under test with a corner case data but analyze the control-flows graph in order to find additional data to fuzz .
```
Comment (alexeyfridman) [inline .github/ISSUE_TEMPLATE/feature_request.md:20]: remove " " at the end

**Review by alexeyfridman — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -7,26 +7,32 @@ assignees: ''
...
-**Alternatives you've considered**
+**Environment**
 
-Substitute this text with a potential alternative to the solution.
+_Substitute the text with steps to set up the environment for testing of the expected behavior._ 
 
-_Example:_ We can use more simple version of a fuzzer that just finds corner cases for primitives.
+_Example:_ You can execute test generation for a complex example. Stop the generation in several seconds after the start.
+
+**Potential alternativesd**
```
Comment (dtim) [inline .github/ISSUE_TEMPLATE/feature_request.md:28]: Typo: "alternativesd", should be "alternatives"

Comment (dtim) [inline .github/ISSUE_TEMPLATE/feature_request.md]: As discussed: should we add a section like "Potential risks and drawbacks" (either before "Potential alternatives" or after "Description"?

Comment (denis-fokin) [reply .github/ISSUE_TEMPLATE/feature_request.md]: This is a very good suggestion.  I will apply it as a separate change.

**Review by dtim — APPROVED** [review]
